workflows/raredisease.nf

```diff
@@ -483,16 +483,9 @@ workflow RAREDISEASE {
                 .groupTuple()
                 .set { ch_merged_vcf }
 
-            ANNOTATE_SNVS.out.tbi
-                .concat(ANALYSE_MT.out.tbi)
-                .groupTuple()
-                .set { ch_merged_tbi }
-
-            ch_merged_vcf.join(ch_merged_tbi, failOnMismatch:true, failOnDuplicate:true).set {ch_concat_in}
-
-            BCFTOOLS_CONCAT (ch_concat_in)
-            ch_snv_annotate = BCFTOOLS_CONCAT.out.vcf
-            ch_versions = ch_versions.mix(BCFTOOLS_CONCAT.out.versions)
+            GATK4_MERGEVCFS (ch_merged_vcf, [[],[]])
```

Comment:
Will the resulting VCF-file be sorted? If not, then should it perhaps be sorted?

Reply:
Good point, Anders! MergeVcfs does generate sorted vcfs. I noticed that it does so in my tests, and now on looking at the tool documentation more closely, it says so in the description as well, https://gatk.broadinstitute.org/hc/en-us/articles/360037226612-MergeVcfs-Picard- :)